src/prob/tp_opf_bf_del_kclmx.jl

```diff
@@ -0,0 +1,68 @@
+""
+function run_tp_opf_bf_del_kclmx(data::Dict{String,Any}, model_constructor, solver; kwargs...)
```

Comment:
the function names are becoming a bit unwieldy, is it possible to simplify them, or make their meaning more clear? at the very least I think a docstring with the name spelled out will make things better for the user. Check other instances.